projects/packages/search/src/dashboard/components/record-meter/bar-chart.jsx

```diff
@@ -146,7 +146,7 @@ export class BarChart extends React.Component {
 									/>
 									<span className="jp-search-chart-legend__label" children={ item.text } />
 									<span className="jp-search-chart-legend__count">
-										({ this.props.data[ item.datasetIndex ].data.data })
+										({ this.props.data[ item.datasetIndex ].data.data.toLocaleString() })
```

Comment:
Do we have any way to ensure that this will be a string? Invoking `toLocaleString` on a non-string will break the render cycle.

Reply:
Ok really good question! I am going to investigate this a little further. I'll merge for now, but going to approach this in a follow-up PR with a couple of other small fixes to things.

Reply:
@annacmc I think optional chaining could be useful here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Correct! To be safest we should check that we're actually dealing with a string before calling a string-specific method on it:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof

You can also use optional chaining multiple times for a deeply-nested structure, like `this.props.data?.something?.otherthing`.